package.json

```diff
@@ -2,6 +2,7 @@
   "name": "mitt",
   "version": "3.0.0",
   "description": "Tiny 200b functional Event Emitter / pubsub.",
+  "type": "module",
   "module": "dist/mitt.mjs",
   "main": "dist/mitt.js",
```

Comment:
This is incorrect and not viable (on its own). Setting the package type to `"module"` means all of those `.js` files will be treated as ESM when they're CJS.